sw/airborne/peripherals/mpu60x0_i2c.c

```diff
@@ -150,10 +153,19 @@ bool_t mpu60x0_configure_i2c_slaves(Mpu60x0ConfigSet mpu_set, void *mpu)
       mpu_i2c->slave_init_status++;
       break;
     case MPU60X0_I2C_CONF_SLAVES_CONFIGURE:
+#ifdef MPU9150_SLV_MAG
+       while(cpt<mpu_i2c->config.nb_slaves) {
```

Comment:
I think this should not be done in a while loop, rather it should call `mpu_i2c->config.slaves[cpt].configure(mpu_set, mpu)` once and move to the next slave if it succeeded.
That also means that `cpt` would need to be static...